### PR TITLE
Perform a sync between Rust and JS when generating markdown instead of in spec tests

### PR DESCRIPTION
**Pull request by maciejhirsz**
So builds stop failing when someone adds a method, but there will be a clear error when running `npm run build:markdown` - which is when you really want to see it (md files will still be generated).

### CODE REVIEW

File: js/scripts/helpers/log.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import chalk from 'chalk';
```
Comment (jacogr) [inline js/scripts/helpers/log.js]: Just the the GPL header here.

File: js/scripts/helpers/parsed-rpc-traits.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import fs from 'fs';
```
Comment (jacogr) [inline js/scripts/helpers/parsed-rpc-traits.js]: Same here, GPL header.

**Comment by jacogr** [comment]
Tiny comments, apart from that looks good.

**Comment by maciejhirsz** [comment]
@jacogr cheers, done

**Pull request merged**